.github/actions/convertIssue/action.yml

```diff
@@ -1,4 +1,5 @@
 # this action is based on the action by zachleat on github (github-issue-to-json-file)
+# https://github.com/zachleat/github-issue-to-json-file/blob/main/action.yml
```

Comment:
@Abby-Wheelis in general, we cannot use copy that does not explicity have a LICENSE. We can "adapt" based on it if the changes are sufficiently large. You might want to check with legal on whether this meets that threshold.